ansible/roles/pyassimp/tasks/main.yml

```diff
@@ -0,0 +1,26 @@
+# Fix pyassimp issue
+
+- name: Create pyassimp directory if it does not exist
+  file:
+     path: /tmp/assimp
+     state: directory
+     mode: 0755
+
+- name: Clone pyassimp source code
+  git:
+    repo: 'https://github.com/assimp/assimp.git'
+    version: 51520cb5f1adcb588a2533d450aace480c7cc3d4
+    dest: "/tmp/assimp"
+    clone: yes
+    depth: 1 
+    update: yes
+
+- name: Build and install
+  shell: bash -c "cmake CMakeLists.txt -G 'Unix Makefiles' && make && make install"
+    chdir=/tmp/assimp
+  sudo: yes
+
+- name: Install
+  shell: bash -c "python setup.py install"
+    chdir=/tmp/assimp/port/PyAssimp
+  sudo: yes
```

Comment:
it would be nice to delete folder /tmp/assimp here if it is not needed after installation to reduce size of docker images